### PR TITLE
libdefs: expo-web-browser; @sentry/react-native

### PR DESCRIPTION
**Pull request by chrisbobbe**
After this and #5317, except for anything unexpected that I haven't accounted for yet, there's just [one react-native-webview thread](https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/react-native-webview.20and.20UIManager/near/1357825) that's a Flow-related thing that I'm aware needs investigation before Flow v0.149 and React Native 65 (#5230)

### CODE REVIEW

File: types/expo-web-browser/WebBrowser.types.js.flow [hunk 1]
```diff
@@ -0,0 +1,94 @@
+/**
+ * Types for the NPM package `expo-web-browser/build/WebBrowser.types`.
```
Comment (gnprice) [inline types/expo-web-browser/WebBrowser.types.js.flow:2]: nit: This isn't actually the name of an NPM package.

What this file does describe is a particular module within the NPM package `expo-web-browser`. Could perhaps say "the NPM module `expo-web-browser/build/WebBrowser.types`", in that it's the unique module that you could import under that name as a result of installing something from NPM.

**Review by gnprice — COMMENTED** [review]
Thanks! Looks good, with small comments below just on comments -- then please merge at will.

File: types/@sentry/react-native.js.flow [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+export type Hub = {
+  getClient(): Client | typeof undefined,
+
+  captureException(exception: mixed, hint?: EventHint): string,
+  captureMessage(message: string, level?: SeverityType, hint?: EventHint): string,
+  ...
+};
+
+// Really an exact object type with the commented-out properties present.
+// (They're omitted lazily.) Taken from
```
[the remark this answers is not in the document]
Comment (chrisbobbe) [reply types/@sentry/react-native.js.flow:153]: Ha, yes. The passive voice was probably adopted to obstruct a "we are lazy" reading, and I should have just said what I meant. Then the reader could decide if the laziness was OK. 😛

**Comment by chrisbobbe** [comment]
Thanks for the review! Revision pushed.

**Comment by gnprice** [comment]
Thanks! Merging.

**Pull request merged**